ddtrace/llmobs/_llmobs.py

```diff
@@ -210,9 +210,9 @@ def _llmobs_span_event(cls, span: Span) -> Tuple[Dict[str, Any], bool]:
             llmobs_span_event["session_id"] = session_id
 
         llmobs_span_event["tags"] = cls._llmobs_tags(
-            span, ml_app, session_id, is_ragas_integration_span=is_ragas_integration_span
+            span, ml_app, session_id, is_ragas_integration_span=is_evaluation_span
```

Comment:
Seems like we don't necessarily need to pass this information in to `_llmobs_tags()` or even return it from this function. Thoughts on just doing the checks individually when needed instead of passing it around?